backend/test/services/index/ElasticsearchEventsITest.scala

```diff
@@ -1,54 +1,73 @@
 package services.index
 
+import com.dimafeng.testcontainers.ElasticsearchContainer
+import com.dimafeng.testcontainers.scalatest.TestContainersForAll
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.should.Matchers
+import services.events._
+import test.AttemptValues
+import test.integration.{ElasticSearchTestContainer, ElasticsearchTestService}
+import utils.attempt.AttemptAwait._
+
 import java.time.Instant
 import java.time.temporal.ChronoUnit
+import scala.concurrent.ExecutionContext
 
-import services.events.{ActionComplete, ActionFailed, ActionStarted, TagEquals, TagNotEquals}
-import test.integration.ElasticsearchTestService
-import utils.attempt.AttemptAwait._
-import org.scalatest.freespec.AnyFreeSpec
-import org.scalatest.matchers.should.Matchers
+class ElasticsearchEventsITest extends AnyFreeSpec with Matchers with TestContainersForAll with ElasticSearchTestContainer {
 
-class ElasticsearchEventsITest extends AnyFreeSpec with Matchers with ElasticsearchTestService {
+  final implicit def executionContext: ExecutionContext = ExecutionContext.global
+
+  override type Containers = ElasticsearchContainer
+
+  var elasticsearchTestService: ElasticsearchTestService = _
+
+  override def startContainers(): Containers = {
+    val elasticContainer = getElasticSearchContainer()
+    val url = s"http://${elasticContainer.container.getHttpHostAddress}"
+
+    elasticsearchTestService = new ElasticsearchTestService(url)
+
+    elasticContainer
+  }
```

Comment:
This could end up time consuming to find a good way of reducing the repetition, so for now lets leave it and if we get a chance we can improve this in an upcoming PR